<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stdDrinkDialog</class>
 <widget class="QDialog" name="stdDrinkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculate Std. Drinks</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="volumeLabel">
     <property name="text">
      <string>Volume:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="abvInputLabel">
     <property name="text">
      <string>ABV:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="standardDrinksLabel">
     <property name="text">
      <string>Standard drinks:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="abvInput"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="standardDrinksOutput">
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="stdDrinkCalcOkButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="volumeInput">
     <property name="decimals">
      <number>1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
